tests/protocol/test_extensions_goal_state.py

```diff
@@ -73,3 +73,16 @@ def test_create_from_vm_settings_should_assume_block_when_blob_type_is_not_valid
         with mock_wire_protocol(data_file) as protocol:
             actual = protocol.client._extensions_goal_state_from_vm_settings.status_upload_blob_type
             self.assertEqual("BlockBlob", actual, 'Expected BlockBob for an invalid statusBlobType')
+
+    def test_create_from_vm_settings_should_parse_requested_version_properly(self):
+        with mock_wire_protocol(mockwiredata.DATA_FILE) as protocol:
+            ga_manifests = protocol.client._extensions_goal_state_from_vm_settings.agent_manifests
+            for manifest in ga_manifests:
+                self.assertIsNone(manifest.version, "Requested version should be None")
```

Comment:
what does None mean? can you document that in VMAgentManifest?

Reply:
None basically means no requested version. I'll add a comment there

Reply:
Done